code/XMLTester.xml

```diff
@@ -710,12 +710,7 @@
 <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
 -->
     <data_writer profile_name="writer_profile_qos_flowcontroller">
-        <!-- NOTE: This tag has been deprecated but doesn't have an
-             equivalent XML tag yet -->
-        <throughputController>
-            <bytesPerPeriod>300000</bytesPerPeriod>
-            <periodMillisecs>1000</periodMillisecs>
-        </throughputController>
+        <!-- TODO: Configure FlowController through XML -->
```

Comment:
The snippet is not used but I left it with the `TODO` so it is known that is something to be done at some point. I am not against of removing the snippet.

(I am for cleaning the C++ code though. There are two files `CodeTester.cp` and `DDSCodeTester.cpp`. Most of the snippets are taken from the DDS, though there are a few coming from the old file. It would be nice to migrate those few that are still only in the older file and remove that file completely :wink:)